lib/build/script_compactor.dart

```diff
@@ -81,11 +81,11 @@ class ScriptCompactor {
 
   Future<Asset> run() {
     var crawler = new ImportCrawler(transform, primaryInput, logger);
-    return crawler.crawlImports().then((imports) {
+    return crawler.crawlImports().then((imports) async {
```

Comment:
`analyzer` wants a `(...)- > Asset` and instead it gets a `(...) -> Future<Asset>`. They shoud change `Future.then` signature to return a `dynamic` ? Should I send an issue to `sdk`  or to `analyzer` to ignore it ?